src/main/java/edu/cornell/kfs/ksr/document/validation/impl/SecurityGroupRule.java

```diff
@@ -87,8 +85,8 @@ protected boolean dataDictionaryValidate(MaintenanceDocument document) {
         SecurityGroup temp = retrieveSecurityGroupByName(securityGroup.getSecurityGroupName());
         if (ObjectUtils.isNotNull(temp)) {
             if (securityGroup.getSecurityGroupId() != null) {
-                if (temp.getSecurityGroupName().equals(securityGroup.getSecurityGroupName())
-                        && (!temp.getSecurityGroupId().equals(securityGroup.getSecurityGroupId()))) {
+                if (StringUtils.equals(temp.getSecurityGroupName(), securityGroup.getSecurityGroupName())
```

Comment:
Same question, would case be an issue with this string compare?

Reply:
Updated to be case-insensitive.